src/pages/index.jsx

```diff
@@ -168,6 +193,23 @@ export const pageQuery = graphql`
         excerpt
       }
     }
+    content: github {
+      repository(owner: "react-knowledgeable", name: "talks") {
```

Comment:
We can also merge this `query` with `talks query` below as well

Reply:
But leave it like this also fine. Maybe it's more clean. XD

Reply:
I'm wondering, how does the merge work? The first content query is meant to be grouped by labels.

Reply:
Sorry, I mean you can do.
```gql
{
  github {
    repository(owner: "react-knowledgeable", name: "talks") {
      content: labels {
         ....
      }
      talks: issues {
         ....
      }
    }
  }
}
```
Which it should request data from github once and got 2 data instead of request 2 times, I guess.

Reply:
updated, thanks for the suggestion :)